Lombiq.VueJs.Samples/Lombiq.VueJs.Samples.csproj

```diff
@@ -42,12 +42,13 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\..\Libraries\Lombiq.HelpfulLibraries\Lombiq.HelpfulLibraries.Common\Lombiq.HelpfulLibraries.Common.csproj" />
+    <ProjectReference Condition="'$(NuGetBuild)' != 'true'" Include="..\..\..\Libraries\Lombiq.HelpfulLibraries\Lombiq.HelpfulLibraries.Common\Lombiq.HelpfulLibraries.Common.csproj" />
```

Comment:
Rather add a conditional `ItemGroup` like in other projects for distinction.

Reply:
Using conditional `ItemGroup` kill the project for me actually:
![image](https://github.com/Lombiq/Orchard-Vue.js/assets/92299130/0ab50949-8cae-4de4-86f4-3bd495448146)

Reply:
It works in dozens of projects, including here a few lines below, so please make sure you're doing it correctly.

Reply:
According to the MSB4232 you can't include something in an `ItemGroup` that doesn not have `Include, Update,Remove`.
We are only trying to `Import Project=` thus why it gives an error and the project won't load.
Neither of the projects have Import elements in ItemGroups.
`<Import>` is actually an inliner in every single usecase in our projects.

Reply:
Lombiq.Hosting.Tenants.FeaturesGuard has exactly what you need:

```
  <ItemGroup Condition="'$(NuGetBuild)' != 'true'">
    <ProjectReference Include="..\..\..\Libraries\Lombiq.HelpfulLibraries\Lombiq.HelpfulLibraries.Common\Lombiq.HelpfulLibraries.Common.csproj" />
    <ProjectReference Include="..\..\..\Libraries\Lombiq.HelpfulLibraries\Lombiq.HelpfulLibraries.OrchardCore\Lombiq.HelpfulLibraries.OrchardCore.csproj" />
  </ItemGroup>
```